docs/includes/_access_token_create_public_client_pkce.mdx

```diff
@@ -1,105 +1,105 @@
-  In this example, the authenticator configuration type is "Hosted Web", so the authentication url is hosted by Beyond Identity (see [Authenticator Configurations](/docs/next/authentication)).  
+  In response to the '/authorize' request, Beyond Identity redirects to the app's configured authentication url.
+  In this example, the authenticator configuration type is "Hosted Web", so the authentication url is hosted by Beyond Identity (see [Authenticator Configurations](/docs/next/authentication)).
 
   :::info important  
   Please note that the redirection must be handled in the front end (such as React, Angular, or vanilla JavaScript running in the user's browser) in order for passkey authentication to work.  
-  :::  
+  :::
 
-  Once passkey based authentication is complete, Beyond Identity will redirect back to the app's configued redirect URI with a 'code' parameter in the query string.  You will use this code in the '/token' request.  
+  Once passkey based authentication is complete, Beyond Identity will redirect back to the app's configued redirect URI with a 'code' parameter in the query string. You will use this code in the '/token' request.
 
+1. Create your '/token' request
 
-1. Create your '/token' request  
+  Create the HTTP request with the following properties:
 
-  Create the HTTP request with the following properties:  
+  **Request url:**
 
-  **Request url:**  
   ```http
   https://auth-{us|eu}.beyondidentity.com/v1/tenants/{tenant_id}/realms/{realm_id}/applications/{application_id}/token
   ```
-  **Request method:** POST  
 
-  **Request headers:**   
+  **Request method:** POST
+
+  **Request headers:**
 
   ```http
-  content-type: application/x-www-form-urlencoded  
+  content-type: application/x-www-form-urlencoded
   ```
 
-  **Request body:**  
+  **Request body:**
+
   ```http
   grant_type: "authorization_code"
   code: {authorization_code}
   client_id: {client_id}
   redirect_uri: {redirect_uri}
-  code_verifier=${codeVerifier}
+  code_verifier: ${codeVerifier}
   expiration_time: {seconds}
   custom_claims: {custom_claims_json}
   ```
 
-  where:  
+  where:
+
+    - {authorization_code} is the code returned from the '/authorize' call
 
-   - {authorization_code} is the code returned from the '/authorize' call  
+    - {client_id} is the app's configured **Client ID**
 
-   - {client_id} is the app's configured **Client ID**    
+    - {redirect_uri} is one of the app's configured **Redirect URI** values and matches the redirect_uri sent in the '/authorize' call
 
-   - {redirect_uri} is one of the app's configured **Redirect URI** values and matches the redirect_uri sent in the '/authorize' call  
+    - {codeVerifier} is defined as in [RFC 7636](https://datatracker.ietf.org/doc/html/rfc7636#section-4.1), example JavaScript snippet below:
 
-   - {codeVerifier} is defined as in [RFC 7636](https://datatracker.ietf.org/doc/html/rfc7636#section-4.1), example JavaScript snippet below:  
-  
```

Comment:
table again :) probably in all of these include files

Reply:
Done 👍